addon/styles/partials/_uni-step-card.scss

```diff
@@ -5,6 +5,34 @@
         width: 100%;
     }
 
+    &--passed {
```

Comment:
Should this be part of our CSS framework (uniq), @mmelo ?

Reply:
not wanting to nitpick, I must agree with @AndreJoaquim, this should be on uniQ (css) as it, if used elsewhere should have the passed state colors according to the rest of the changes made here. 🎖